api4/channel.go

```diff
@@ -1717,6 +1735,8 @@ func addChannelMember(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	auditRec.Success()
+	auditRec.AddEventResultState(cm) // TODO what about the channel object? what's prior and resulting state... and type?
```

Comment:
I think the object_type you set makes sense. I don't think there is a prior state as this is a create operation, for resulting state that would be the channel membership (which we may not have here as a result and can be added later)